app/default/data.py

```diff
@@ -229,7 +229,21 @@ def get_round_data_fail_gracefully(fund_id, round_id):
         )
         # return valid Round object with no values so we know we've
         # failed and can handle in templates appropriately
-        return Round("", [], "", "", "", "", "", "", "", "", {}, {})
+        return Round(
+            id="",
+            assessment_criteria_weighting=[],
+            assessment_deadline="",
+            deadline="",
+            fund_id="",
+            opens="",
+            title="",
+            short_name="",
+            prospectus="",
+            privacy_notice="",
+            instructions="",
+            contact_details={},
+            support_availability={},
+        )
```

Comment:
Maybe we could make this a class method of round, so it could be easily re-used elsewhere if need be?

i.e in https://github.com/communitiesuk/funding-service-design-frontend/blob/3e02a7f54fd546d934ba858ab3de4454018e1826/app/models/round.py#L58
```python
@classmethod
@property
def empty_round() -> Round:
    return Round(
            id="",
            assessment_criteria_weighting=[],
            assessment_deadline="",
            deadline="",
            fund_id="",
            opens="",
            title="",
            short_name="",
            prospectus="",
            privacy_notice="",
            instructions="",
            contact_details={},
            support_availability={},
        )
```

```suggestion
        return Round.empty_round
```